site/data/ytt/docs/ytt-develop-toc.yml

```diff
@@ -45,6 +45,8 @@ toc:
         url: /lang-ref-ytt
       - page: Struct module
         url: /lang-ref-ytt-struct
+      - page: Assert module
+        url: /lang-ref-ytt-assert
```

Comment:
Similar point as above about alphabetizing.

It currently feels like less of an issue because there's not yet more than seven (7).

Also, being alphabetical can help convey that this section is reference material.